tests/e2e/specs/add-edit-category.test.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	pressKeyWithModifier,
+	pressKeyTimes
+} from '@wordpress/e2e-test-utils'
+import { addQueryArgs } from '@wordpress/url';
+
+describe( 'Categories tests', () => {
+	const title = 'New Category';
+	const query = addQueryArgs( '', {
+		taxonomy: 'category',
+	} );
+
+	beforeEach( async () => {
+		/**
+		 * Delete all categories before anything
+		 * This is useful because after running more than one test
+		 * there could be existing categories
+		 */
+		await visitAdminPage( 'edit-tags.php', query );
+		await page.$( '#bulk-action-selector-top' );
+		await page.waitForSelector( '[id^=cb-select-all-]' );
```

Comment:
Do we need to await for both `#bulk-action-selector-top` and the select all checkbox here?

Reply:
Not really.

I have to admit that I added most of the `waitForSelector` as an extra safeguard, but we can do without as well.